Simplenote/AuthViewController+Swift.swift

```diff
@@ -201,6 +201,22 @@ extension AuthViewController {
 
         alert.beginSheetModal(for: window, completionHandler: completion)
     }
+
+    @objc
+    func showUnverifiedEmailAlert(for window: NSWindow, completion: @escaping (NSApplication.ModalResponse) -> Void) {
+        let alert = NSAlert()
+        alert.messageText = Localization.unverifiedMessageText
+        alert.informativeText = Localization.unverifiedInformativeText
+        alert.addButton(withTitle: Localization.unverifiedActionText)
+        alert.addButton(withTitle: Localization.unverifiedCancelText)
+
+        alert.beginSheetModal(for: window, completionHandler: completion)
+    }
+
+    @objc
+    func sendVerificationMessage(for email: String) {
+        AccountVerificationRemote().verify(email: email) { _ in }
```

Comment:
We should probably replicate what @eshurakov mentioned [here](https://github.com/Automattic/simplenote-ios/pull/1398#discussion_r690185403). Pasting for convenience!!

```
I think we should show an activity indicator (can use the one we have for account deletion) and handle errors similar to what we're doing in AccountVerificationViewController
```

Reply:
Good idea.  I have added the check to confirm that the 403 error is "requires verification" and added a ProgressIndicator